accounts.csv

```diff
@@ -0,0 +1,141 @@
+regen1j5az0zsgacwks4v5qvlwpqycqvgw3nlsmgg2jy,10869.56522,2022-02-06,24
+regen1j5az0zsgacwks4v5qvlwpqycqvgw3nlsmgg2jy,100315.4444,2024-02-06,1
+,137388.5435,2022-02-06,24
+regen1uygmm78q793l90vem2xvr0fx5tvmt6krjz9w57,543478.2609,2022-02-07,24
+regen1jzqfg3rq8hdav6wfcmrdk5l47y8wdwxx7c4pfa,17391.30435,2022-02-08,24
+regen182ckcndusy25lng570864xtk25whk6ptkf4j2g,21739.13043,2022-02-06,24
+regen1tv2edevd60qj0rc0zzmut26l868xdd9muqufx9,21739.13043,2022-02-10,24
+,31746.03175,2022-02-09,1
+regen1984ym33nggaaucj7pcec4a6ywrxw8lq5df7a92,8790.47619,2024-02-11,1
+regen1u26sz6kydnmpr5zq6nureav4xhasak8n5cxds8,2173.913043,2022-02-11,24
+regen1vwnvwkhkz9m2rr3d2al0ggmaff587awj4cptke,31746.03175,2022-02-11,1
+regen1rymfw64cg4ffkjy8ls8anv3622g4lcgtyz0l4a,2380.952381,2024-02-11,1
+regen189cq3yr0nsgr67cp7q6q0gj4xv9rtzkkpv3fyu,2177.543478,2022-02-11,24
+regen10valuxrrt5guceq5eu8tcf566vkegylkanzf68,41304.34783,2022-02-10,24
+regen1j4vc4arytknz9uvhcykzsazctz58lzef0hrmld,21739.13043,2022-02-10,24
+regen1j5az0zsgacwks4v5qvlwpqycqvgw3nlsmgg2jy,1587.301587,2022-02-12,1
+regen1sw0gdwv5e6qhpz62cr0gr0xa532rhhmp67jasq,53058.5873,2022-02-12,1
+regen1sl3jy9580xdpqzy645l5f0r3p83evljva2rkzw,1587.301587,2022-02-11,1
+regen1sl3jy9580xdpqzy645l5f0r3p83evljva2rkzw,1587.301587,2024-02-11,1
+regen1xhhquwctvwm40qn3nmmqq067pc2gw22e6q3p3c,2173.913043,2022-02-11,24
+regen1xhhquwctvwm40qn3nmmqq067pc2gw22e6q3p3c,4761.904762,2024-02-12,1
+regen1yqvxyhzqd4e6mg6hhd0x08jr5yg6mdzjpuvv9w,28260.86957,2022-02-12,24
+,2823.26087,2022-02-12,24
+regen12p4dwxd4zusmz2c5r7za8nzcjkxl3wthmqex9a,10869.56522,2022-02-13,24
+regen1jzqfg3rq8hdav6wfcmrdk5l47y8wdwxx7c4pfa,10869.56522,2022-02-13,24
+regen17dqm2ud9kjy6ue4j2exgjxfyg9nj32q0nxh330,10873.95652,2022-02-13,24
+regen1sznvffkrlr3eqq4mfl90j56v0x47f2t2r8a40x,15873.01587,2022-02-11,1
+,14285.71429,2024-02-14,1
+regen1eef3zlxmge0jnkp3vrwkn3csuyze8eyg6f3aal,6521.73913,2022-02-14,24
+regen1dp5evmfuf4fwx295w049s99nyfz29fvttj0wc3,3174.603175,2024-02-15,1
+regen1lyv3cgtldkcrwahh0xqt6lw4hp28k89mmed3gv,30021.73913,2022-02-15,24
+regen1mn4c8kwzm9uvtquzrazr23w0pcdwdkcm7psp4f,2173.913043,2022-02-15,24
+regen16fl09ams6v2ktwhrsn5ghpmlgexwqf5a3lj7kz,5555.555556,2022-02-10,1
+regen18cu95qe7s58ard3nxcf9newkvty7l0c0ulf0xh,3968.253968,2024-02-14,1
+regen1x7a0ltsxnpx9y3rt7s45q33a83nv7dlfgvryww,108695.6522,2022-02-14,24
+regen1dy69x4ls39u5kmmu7tcjye9jnrmh6g6mzrmcsz,15873.01587,2022-02-15,1
+regen1qryzcujn658shgk74nlqh93qpaqlfkadq26k2v,3174.603175,2022-02-16,1
+regen1l9sdckges543mvujcwwqaszfusn43ul42u43x7,4761.904762,2024-02-16,1
+regen1l9sdckges543mvujcwwqaszfusn43ul42u43x7,6521.73913,2022-02-16,24
+regen17kav8azytka7486y92e6tewk2sh6r8er98un9f,3174.603175,2024-02-16,1
+regen1afllqnt53t7rapl2v67rd93ww9e07g47qafanz,5673.913043,2022-02-16,24
+regen13yd53zavqcfrzml6jtaf8n8se8q4lcyqt8qqsz,79365.07937,2024-02-15,1
+regen1lehlhuapmw5f9f0q3uyctne4x59klh35s8v04v,54347.82609,2022-02-15,24
+regen150tcpahw4m80aysv54qua6unysp9ya55la9lzk,43478.26087,2022-02-16,24
```

Comment:
Can we verify that the import script will successfully read the last line without a newline? Or should we just add a `\n` character to be safe?